arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -4594,7 +4594,6 @@
 			buildPhases = (
 				3E23A9DC1AFC28F6002E2214 /* Sources */,
 				3E23A9DD1AFC28F6002E2214 /* Frameworks */,
-				4CB506D1224C2C20006A3471 /* Download Portal Item Data */,
```

Comment:
By removing the build phase, the output of download swift script is written to stdout/stderr instead, which means we cannot see it in the Xcode build log. We may need to find other ways to display these information, such as using notifications.